docfiles/docs.js

```diff
@@ -47,8 +54,21 @@ function searchSubmit(form) {
 }
 
 function setupSidebar() {
+    let togglesidebar = document.getElementById("togglesidebar");
+    togglesidebar.onkeydown = handleEnterKey;
```

Comment:
Yeah let's stay away from JQuery, there's no need to have react and JQuery. This isn't as relevant for v0, but since this PR would be going to master, I want to stay away from JQuery.